src/modules/catalog.js

```diff
@@ -1822,7 +1833,7 @@ class Catalog {
    *         pattern: 'outerwear',
    *         match_type: 'EXACT'
```

Comment:
Should `match_type` also be changed to be camelCase? 🤔 I'm not sure how far we want to take this